lib/gui/modules/analytics.js

```diff
@@ -22,106 +22,44 @@
-const os = require('os');
 const packageJSON = require('../../../package.json');
-const arch = require('arch');
-const utils = require('../../shared/utils');
 
-// Force Mixpanel snippet to load Mixpanel locally
-// instead of using a CDN for performance reasons
-window.MIXPANEL_CUSTOM_LIB_URL = '../../bower_components/mixpanel/mixpanel.js';
-
-require('../../../bower_components/mixpanel/mixpanel-jslib-snippet.js');
-require('../../../bower_components/angular-mixpanel/src/angular-mixpanel');
 const MODULE_NAME = 'Etcher.Modules.Analytics';
 const analytics = angular.module(MODULE_NAME, [
-  'analytics.mixpanel',
   require('../models/settings')
 ]);
 
-/**
- * @summary Get host architecture
- * @function
- * @private
- *
- * @description
- * We need this because node's os.arch() returns the process architecture
- * See: https://github.com/nodejs/node-v0.x-archive/issues/2862
- *
- * @returns {String} Host architecture
- *
- * @example
- * if (getHostArchitecture() === 'x64') {
- *   console.log('Host architecture is x64');
- * }
- */
-const getHostArchitecture = () => {
-  return _.includes([ 'ia32', 'x64' ], process.arch) ? _.replace(arch(), 'x86', 'ia32') : process.arch;
-};
-
-// Mixpanel integration
-// https://github.com/kuhnza/angular-mixpanel
-
-analytics.config(($mixpanelProvider) => {
-  $mixpanelProvider.apiKey('63e5fc4563e00928da67d1226364dd4c');
-
-  $mixpanelProvider.superProperties({
-    electron: process.versions.electron,
-    node: process.version,
-    arch: process.arch,
-    version: packageJSON.version,
-    osPlatform: os.platform(),
-    hostArch: getHostArchitecture(),
-    osRelease: os.release(),
-    cpuCores: os.cpus().length,
-    totalMemory: os.totalmem(),
-    startFreeMemory: os.freemem()
-  });
-});
-
-// TrackJS integration
-// http://docs.trackjs.com/tracker/framework-integrations
-
-analytics.run(($window) => {
-
-  // Don't configure TrackJS when
-  // running inside the test suite
-  if (window.mocha) {
-    return;
-  }
-
-  $window.trackJs.configure({
-    userId: username.sync(),
-    version: packageJSON.version
+analytics.config(() => {
+  resinRaven.install({
+    services: {
+      sentry: process.env.SENTRY_DSN,
+      mixpanel: process.env.MIXPANEL_TOKEN
+    },
+    options: {
+      release: packageJSON.version
+    }
   });
 });
 
-analytics.service('AnalyticsService', function($log, $window, $mixpanel, SettingsModel) {
+analytics.service('AnalyticsService', function($log, SettingsModel) {
 
   /**
    * @summary Log a debug message
    * @function
    * @public
    *
    * @description
-   * This function sends the debug message to TrackJS only.
+   * This function sends the debug message to Sentry only.
    *
    * @param {String} message - message
    *
    * @example
    * AnalyticsService.log('Hello World');
    */
   this.logDebug = (message) => {
-    const debugMessage = `${new Date()} ${message}`;
-
-    if (SettingsModel.get('errorReporting') && isRunningInAsar()) {
```

Comment:
It wasn't intended. I did this now: https://github.com/resin-io/etcher/pull/1208/files#diff-05cd97d7d3334a665e1467c4c5d81043R90, let me know what you think. I do need to better test it though.